code/controllers/configuration/entries/game_options.dm

```diff
@@ -174,10 +174,22 @@
 /datum/config_entry/string/alert_delta
 	default = "Destruction of the station is imminent. All crew are instructed to obey all instructions given by heads of staff. Any violations of these orders can be punished by death. This is not a drill."
 
-//MONKESTATION EDIT START
+// monkestation start: additional alert levels
 /datum/config_entry/string/alert_epsilon
 	default = "Central Command has ordered the Epsilon security level on the station. Consider all contracts terminated."
-//MONKESTATION EDIT STOP
+
+/datum/config_entry/string/alert_yellow
+	default = "The security alert level has been changed to yellow. At this time, severe structural, atmospheric, or otherwise engineering damage has occurred to %STATION_NAME%. Crewmembers are advised to seek shelter, and Engineering are given enhanced powers for the duration of this alert; including immunity to trespass, breaking and entering, and theft charges committed in service to remedying the danger."
```

Comment:
Can you change the desc to be instead "The security alert level has been changed to yellow. There is currently a critical Engineering issue on board %STATION_NAME%. Crew members are advised to avoid damaged area's, and report them as needed." ?

Reply:
Done.